forms-flow-web/src/components/Modeler/Editors/BpmnEditor/index.js

```diff
@@ -161,7 +164,11 @@ export default React.memo(
       // Deployment Source
       form.append("deployment-source", "Camunda Modeler");
       // Tenant ID
-      if (tenantKey && !applyAllTenants) {
+      if (tenantKey && !applyAllTenants && publicWorkflowEnabled === "true") {
```

Comment:
Not an UI expert, but is it a good practice to use `===` with string value of boolean ?

Reply:
done changes

Reply:
changed